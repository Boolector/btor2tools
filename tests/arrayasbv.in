1 array 8 32 mem
2 var 8 x
3 add 8 1 2
