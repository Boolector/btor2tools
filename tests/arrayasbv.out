*** catbtor: parse error in 'arrayasbv.in' line 3: expected bit-vector argument
